POGOLib.Official/Net/Session.cs

```diff
@@ -115,16 +114,12 @@ internal Session(ILoginProvider loginProvider, AccessToken accessToken, GeoCoord
         public async Task<bool> StartupAsync()
         {
             if (!Configuration.IgnoreHashVersion)
-            {
-                await CheckHasherVersion();
-            }
+                CheckHasherVersion().Wait();
 
-            if (!await RpcClient.StartupAsync().ConfigureAwait(false))
-            {
+            if (!RpcClient.StartupAsync().Result)
                 return false;
-            }
 
-            await _heartbeat.StartDispatcher().ConfigureAwait(false);
+            _heartbeat.StartDispatcher().Wait();
```

Comment:
because await is to do it syncronous, configureawait(false) to use same threadspace.
so you can replace it using the method .wait(), (or .Resut if return something) leaving the code cleaner.